test/Microsoft.AspNet.Mvc.Core.Test/Filters/FormatFilterTest.cs

```diff
@@ -0,0 +1,253 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Mvc.Core.Filters;
+using Microsoft.AspNet.PipelineCore;
+using Microsoft.AspNet.Routing;
+using Microsoft.Net.Http.Headers;
+using Xunit;
+
+#if ASPNET50
+using Moq;
+using Microsoft.Framework.OptionsModel;
+using Microsoft.AspNet.Http;
+#endif
+
+namespace Microsoft.AspNet.Mvc.Core.Test
+{
+    public enum FormatPlace
+    {
+        RouteData,
+        QueryData,
+        RouteAndQueryData
+    }
+
+    public class FormatFilterTests
+    {
+        [Theory]
+        [InlineData("json", FormatPlace.RouteData, "application/json")]
+        [InlineData("json", FormatPlace.QueryData, "application/json")]
+        [InlineData("json", FormatPlace.RouteAndQueryData, "application/json")]
+        public void FormatFilter_ContextContainsFormat_DefaultFormat(string format, 
```

Comment:
`string format` move to the next line to be consistent with the formatting on the others